cli/README.md

```diff
@@ -22,14 +22,69 @@
 
 ## Installing OnionShare CLI
 
-First, make sure you have `tor` installed. In Linux, install it through your package manager. In macOS, install it with [Homebrew](https://brew.sh): `brew install tor`.
+First, make sure you have `tor` and `python3` installed. In Linux, install it through your package manager. In macOS, install it with [Homebrew](https://brew.sh): `brew install tor`. Second, OnionShare is written in python, and you can install the command line version use python's package manager `pip`.
 
-Then install OnionShare CLI:
+### Requirements
 
+Debian/Ubuntu (APT):
 ```sh
-pip install onionshare-cli
+sudo apt-get install tor python3-pip
 ```
 
+Arch (Pacman):
+```sh
+sudo pacman -S tor python-pip
+```
+
+CentOS, Red Hat, and Fedora (Yum):
+```sh
+sudo yum install tor python3 python3-wheel
+```
+
+macOS (Homebrew):
+```sh
+brew install tor python
+sudo easy_install pip
+```
+
+### Main
+
+#### Installation
+
+Install OnionShare CLI:
+
+```sh
+pip install --user onionshare-cli
+```
+
+#### Set path
+
+When you install programs with pip and use the --user flag, it installs them into ~/.local/bin, which isn't in your path by default. To add ~/.local/bin to your path automatically for the next time you reopen the terminal or source your shell configuration file.
```

Comment:
Can we add at the end of this sentence, `, do the following:`, so that it reads like this?

> To add ~/.local/bin to your path automatically for the next time you reopen the terminal or source your shell configuration file, do the following:

Reply:
done